buildenv/jenkins/variables/defaults.yml

```diff
@@ -250,6 +255,21 @@ ppc64le_linux_xl_cm:
 ppc64le_linux_xl_uma:
   extends: ['ppc64le_linux_xl', 'uma']
 #========================================#
+# Linux PPCLE 64bits Mixed References
+#========================================#
+ppc64le_linux_mixed:
+  extends: ['ppc64le_linux', 'mixedrefs']
+  excluded_tests:
+    8:
+      - special.system
+    11:
+      - special.system
```

Comment:
Why define configurations `*_mixed` and `*_mixed_cm` when only `*_mixed_cm` is supported?

Reply:
I noticed the other configurations followed a similar hierarchy, something like:
- `*`, `*_cm`, `*_uma` for the compressed refs builds
- `*_xl`, `*_xl_cm`, `*_xl_uma` for the large heap builds

so I defined `*_mixed` and I have `*_mixed_cm` extending them.

Although these configs are defined in `buildenv/jenkins/variables/defaults.yml`, they're not exposed in `buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy`. Once the `uma` changes are in, the `*_mixed_uma` configurations will extend the `*_mixed` equivalents as well, and the default `*_mixed` and `*_mixed_uma` will be made available in the pipeline SPEC as well.

Reply:
I didn't realize we planned to support UMA builds - that make sense now.